test/integration-server/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const TEST_SERVER_URL = process.env.TEST_SERVER_URL ?? ''; /* Add here your own test server URL */
```

Comment:
Maybe we should call this file `env-config.ts` or something instead of constants. What do you think?

Reply:
Good idea. Renamed.